test/Microsoft.AspNetCore.Razor.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -363,6 +363,7 @@ public static TheoryData<string> EmptyOrWhiteSpaceData
             {
                 return new TheoryData<string>
                 {
+                    null,
```

Comment:
Used in multiple `[Theory]` tests. 5 of those 9 tests fail without the src changes.